### PR TITLE
Add OB as partner + remove emmanuel espias

### PR DESCRIPTION
**Pull request by glalloue**

### CODE REVIEW

File: public/img/organizations/orange-business.webp
Comment (utarwyn) [inline public/img/organizations/orange-business.webp:1]: Can you resize this image to 160x55? (as described in [src/assets/data/partner-organizations.json](https://github.com/green-code-initiative/ecoCode-website/pull/24/files#diff-5f59b1ba4cd9685caf4aeef9494085887aa59d8850c9268fecf3f33956bb35fa))

File: public/img/profil/jerome_cardon.webp
Comment (utarwyn) [inline public/img/profil/jerome_cardon.webp:1]: Can you resize this image to 80x80?

**Review by utarwyn — CHANGES_REQUESTED** [review]

File: public/img/organizations/orange-business.webp
Comment (glalloue) [reply public/img/organizations/orange-business.webp:1]: yes you're right, i'm going to do that ;)

Comment (glalloue) [reply public/img/organizations/orange-business.webp:1]: image reisized

File: public/img/profil/jerome_cardon.webp
Comment (glalloue) [reply public/img/profil/jerome_cardon.webp:1]: image resized

**Review by utarwyn — APPROVED** [review]
LGTM 💯 Thank you for the changes!

**Pull request merged**